dev-tools/omdb/src/bin/omdb/nexus.rs

```diff
@@ -374,6 +378,51 @@ enum ClickhousePolicyMode {
     Both,
 }
 
+#[derive(Debug, Args)]
+struct ChickenSwitchesArgs {
+    #[command(subcommand)]
+    command: ChickenSwitchesCommands,
+}
+
+#[derive(Debug, Subcommand)]
+enum ChickenSwitchesCommands {
+    /// Show a chicken switch at a given version
+    Show(ChickenSwitchesShowArgs),
+
+    /// Set the value of all chicken switches for the latest version
+    /// Values carry over from the latest version if unset on the CLI.
+    Set(ChickenSwitchesSetArgs),
+}
+
+#[derive(Debug, Clone, Args)]
+struct ChickenSwitchesSetArgs {
+    planner_enabled: bool,
+}
+
+#[derive(Debug, Clone, Copy, Args)]
+struct ChickenSwitchesShowArgs {
+    version: ChickenSwitchesVersionOrCurrent,
+}
+
+#[derive(Debug, Clone, Copy)]
+enum ChickenSwitchesVersionOrCurrent {
+    Current,
+    Version(u32),
+}
+
+impl FromStr for ChickenSwitchesVersionOrCurrent {
+    type Err = ParseIntError;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        if matches!(s, "current" | "latest") {
```

Comment:
nit, take it or leave it: perhaps this should be case insensitive?

Reply:
If you are typing in caps I question your sanity.

Reply:
yeah that's fair, carry on